.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -27,6 +27,12 @@ fun shouldBeSkipped(subProject: GradleSubproject, testConfig: TestCoverage): Boo
     return testConfig.os.ignoredSubprojects.contains(subProject.name)
 }
 
+val cleanDaemonsBeforePerformanceTest = """
+    free -m
+    ps aux | egrep 'Gradle(Daemon|Worker)' | awk '{print ${'$'}2}' | xargs kill -9
+    free -m
```

Comment:
is this meant to assess the memory impact of cleaning the leftovers?
Otherwise it may suffice to keep only the second one

Reply:
> is this meant to assess the memory impact of cleaning the leftovers?

Yes, that's the intention.